package.json

```diff
@@ -72,19 +89,5 @@
       "eslint 'src/**/*.ts' --fix"
     ]
   },
-  "dependencies": {
-    "graphql": "^15.4.0",
-    "graphql-request": "^3.4.0"
-  },
-  "ava": {
-    "files": [
-      "test/**/*"
-    ],
-    "extensions": [
-      "ts"
-    ],
-    "require": [
-      "ts-node/register"
-    ]
-  }
+  "proxy": "https://www.ratemyprofessors.com"
```

Comment:
sorry, not clear on why this is needed

Reply:
Okay, so the whole reason I stumbled upon your project is because I am trying to integrate RMP into my project called https://courseer.co/. When I tried to use the package, I get the CORS problem and after looking up online, one way to fix the issue is by adding a proxy apparently. I am not sure how exactly it fixes it, but it can help with the CORS problem when implementing your package in frontend code to retrieve ratings of the professor. If there is a better way, feel free to explain for both the package or my React project.

Reply:
I believe `proxy` is generally only used for create react apps, I don't think it has a meaning outside of that

Reply:
Interesting, but if I install your package, then it can remove the CORS issue for React apps, right? If it can remove it, then I would prefer keeping it for anyone to use in the future.

Reply:
can you link to documentation to explain how this would help?